pkg/daemon/ceph/osd/daemon_test.go

```diff
@@ -223,3 +223,39 @@ NAME="sdb1" SIZE="30" TYPE="part" PKNAME="sdb"`, nil
 	assert.Equal(t, -1, mapping.Entries["rdb"].Data)
 	assert.Equal(t, -1, mapping.Entries["nvme01"].Data)
 }
+
+func TestGetRemovedDevices(t *testing.T) {
+	testGetRemovedDevicesHelper(t, &rookalpha.StoreConfig{StoreType: config.Bluestore})
+	testGetRemovedDevicesHelper(t, &rookalpha.StoreConfig{StoreType: config.Filestore})
+}
+
+func testGetRemovedDevicesHelper(t *testing.T, storeConfig *rookalpha.StoreConfig) {
+	configDir, _ := ioutil.TempDir("", "")
+	defer os.RemoveAll(configDir)
+	os.MkdirAll(configDir, 0755)
+	nodeName := "node3391"
+	agent, _, _ := createTestAgent(t, "none", configDir, nodeName, storeConfig)
+
+	// mock the pre-existence of osd 1 on device sdx
+	_, _, _ = mockPartitionSchemeEntry(t, 1, "sdx", &agent.storeConfig, agent.kv, nodeName)
+
+	// get the removed devices for this configuration (note we said to use devices "none" above),
+	// it should be osd 1 on device sdx
+	scheme, mapping, err := getRemovedDevices(agent)
+	assert.Nil(t, err)
+	assert.Equal(t, 1, len(mapping.Entries))
+	assert.Equal(t, 1, len(scheme.Entries))
+	logger.Infof("scheme: %+v", scheme)
```

Comment:
is the test logging still needed?

Reply:
nope, this test logging isn't needed anymore. it's leftover from debugging the test originally.  i'll remove it.